### PR TITLE
[MIG][12.0] letsencrypt

### PR DESCRIPTION
**Pull request by thomaspaulb**
Retry of https://github.com/OCA/server-tools/pull/1883

(it needs fixing, but `janverb` is no longer working with us and so we cannot push to the branch)

### CODE REVIEW

**Comment by thomaspaulb** [comment]
@dreispt @NL66278 Could you review this again? I changed the last commit as compared to https://github.com/OCA/server-tools/pull/1883, explanation of the method in https://github.com/OCA/server-tools/pull/2014#issuecomment-1214302135

File: letsencrypt/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# © 2016 Therp BV <http://therp.nl>
```
Comment (NL66278) [inline letsencrypt/__init__.py:1]: Delete copyright lines from __init__.py files

File: letsencrypt/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (NL66278) [inline letsencrypt/__init__.py:2]: And these kind of lines should be `https:`

File: letsencrypt/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Let's Encrypt",
```
Comment (NL66278) [inline letsencrypt/__manifest__.py:4]: Missing website key

**Review by NL66278 — APPROVED** [review]
:+1: LGTM And known to work well. Some nags, but not blocking.

**Comment by thomaspaulb** [comment]
Seeing as other people already confirmed the previous PR, merging this.

**Pull request merged**